csharp/src/Ice/SocketStream.cs

```diff
@@ -12,6 +13,14 @@ namespace ZeroC.Ice
     /// There's an instance of this class for each active stream managed by the multi-stream socket.</summary>
     public abstract class SocketStream : IDisposable
     {
+        /// <summary>A delegate used to send data from a System.IO.Stream value.</summary>
+        public static readonly Action<SocketStream, System.IO.Stream, CancellationToken> IceSendDataFromIOStream =
```

Comment:
Why give an Ice prefix to these 2 static fields?

Reply:
To be consistent with the InputStream/Output readers/writers but it's possible I missed the reason why it's needed there and that it's not needed here. Why do we use Ice prefixes on the readers/writers of Input/OutputStream?